src/azure-cli/azure/cli/command_modules/keyvault/_completers.py

```diff
@@ -8,7 +8,7 @@
 
 
 def _get_token(cli_ctx, server, resource, scope):  # pylint: disable=unused-argument
-    return Profile(cli_ctx=cli_ctx).get_login_credentials(resource)[0]._token_retriever()  # pylint: disable=protected-access
+    return Profile(cli_ctx=cli_ctx).get_raw_token(resource)[0]
```

Comment:
This `_get_token` never works because it doesn't comply with `KeyVaultAuthentication`'s `authorization_callback` signature. 

Also defer to @houk-ms to fix it in the future.